tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
Is this supposed to be a private variable? PS convention is to just make all variables public; this probably shouldn't be an exception; would help with hotpatch-debugging if we could actually see what the value of this variable was.